play/src/main/scala/org/lyranthe/prometheus/client/integration/play/filters/PrometheusFilter.scala

```diff
@@ -0,0 +1,65 @@
+package org.lyranthe.prometheus.client.integration.play.filters
+
+import com.google.inject.{Inject, Singleton}
+import org.lyranthe.prometheus.client._
+import play.api.mvc._
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success, Try}
+
+@Singleton
+class PrometheusFilter @Inject()()(implicit val registry: Registry,
```

Comment:
You suggested this Registry wouldn't be implicit, and would be injected with Guice?